tests/integration/golang/mlflow/artifact/get_artifact_local_test.go

```diff
@@ -0,0 +1,134 @@
+//go:build integration
+
+package artifact
+
+import (
+	"context"
+	"fmt"
+	"io/fs"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/hetiansu5/urlquery"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/api/request"
+	"github.com/G-Research/fasttrackml/pkg/api/mlflow/dao/models"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/fixtures"
+	"github.com/G-Research/fasttrackml/tests/integration/golang/helpers"
+)
+
+type GetArtifactLocalTestSuite struct {
+	suite.Suite
+	runFixtures        *fixtures.RunFixtures
+	serviceClient      *helpers.HttpClient
+	experimentFixtures *fixtures.ExperimentFixtures
+}
+
+func TestGetArtifactLocalTestSuite(t *testing.T) {
+	suite.Run(t, new(GetArtifactLocalTestSuite))
+}
+
+func (s *GetArtifactLocalTestSuite) SetupTest() {
+	s.serviceClient = helpers.NewMlflowApiClient(helpers.GetServiceUri())
+
+	experimentFixtures, err := fixtures.NewExperimentFixtures(helpers.GetDatabaseUri())
+	assert.Nil(s.T(), err)
+	s.experimentFixtures = experimentFixtures
+
+	runFixtures, err := fixtures.NewRunFixtures(helpers.GetDatabaseUri())
+	assert.Nil(s.T(), err)
+	s.runFixtures = runFixtures
+}
+
+func (s *GetArtifactLocalTestSuite) Test_Ok() {
+	defer func() {
+		assert.Nil(s.T(), s.experimentFixtures.UnloadFixtures())
+	}()
+
+	testData := []struct {
+		name   string
+		prefix string
+	}{
+		{
+			name:   "TestWithFilePrefix",
+			prefix: "file://",
+		},
+		{
+			name:   "TestWithoutPrefix",
+			prefix: "",
+		},
+	}
+
+	for _, tt := range testData {
```

Comment:
I mean that you have 2 test cases: `TestWithFilePrefix` and `TestWithoutPrefix` and for both of them you create `experiment`, `run` etc, so why we can create everything before and only once?

Reply:
This variable is encoded into the Experiment.ArtifactLocation so it has to be created in the test loop.